src/block_cache/block_chunkcache.c

```diff
@@ -997,6 +1082,19 @@ __wt_chunkcache_teardown(WT_SESSION_IMPL *session)
     return (ret);
 }
 
+/*
+ * __wt_chunkcache_salvage --
+ *     Remove any knowledge of any extant chunk cache metadata. We can always rebuild the cache
+ *     later, so make no attempt at a "real" salvage.
+ */
+int
```

Comment:
shall we make this inline, given it is a small function?

Reply:
I don't see any need - we do `inline` in headers for visibility reasons, but beyond that I trust the compiler to make that call.